rgw/v2/tests/s3_swift/test_bucket_lifecycle_object_expiration.py

```diff
@@ -147,6 +147,18 @@ def test_exec(config, ssh_con):
                 )
             )
             prefix = prefix if prefix else ["dummy1"]
+            if config.lc_reshard_bucket:
+                cmd = utils.exec_shell_cmd(
+                    f"radosgw-admin bucket reshard --bucket {bucket.name} --num-shards 211"
+                )
+                op = utils.exec_shell_cmd(
+                    "radosgw-admin bucket stats --bucket=%s" % bucket.name
+                )
+                json_doc = json.loads(op)
+                shards = json_doc["num_shards"]
+                log.info(
+                    f"num_shards for bucket {bucket.name} after reshard are {shards}"
+                )
```

Comment:
I think we need to raise test failure exception for this code

Reply:
done

Reply:
@udaysk23 , Having a test failure condition would have the code include a sleep time of maybe 30 seconds or more since resharding to 211 shards takes time. Are we in for increasing the execution time?